misc/lint.sh

```diff
@@ -41,6 +41,7 @@ lint_html () {
   html5validator --root "$SITE_DIR" --show-warnings --ignore-re \
     'Illegal character in query: "\|" is not allowed' \
     '"(autocorrect|autocapitalize)" not allowed on element "input"' \
+    '"autocomplete" is only allowed when the input type is' \
```

Comment:
Is there anything left unsaid?

Reply:
There are a lot of `input` types in the relevant error message (see [this Travis log](https://travis-ci.org/exonum/exonum-doc/builds/358907689)), so I've skipped them.